backend/workflow_manager/endpoint/destination.py

```diff
@@ -153,6 +153,8 @@ def handle_output(
                 file_name=file_name, error=error, result=result, meta_data=meta_data
             )
         elif connection_type == WorkflowEndpoint.ConnectionType.MANUALREVIEW:
+            if file_history:
+                return
```

Comment:
NIT: Consider adding an INFO / DEBUG log indicating that we're not returning anything or that we've already processed the result. Adding more context in the log such as the result itself or the result size will only help us debug issues in staging / prod

Reply:
done adding logger info with workflowID